k2/python/csrc/dlpack.h

```diff
@@ -70,7 +70,7 @@ typedef struct {
   /*! \brief The device type used in the device. */
   DLDeviceType device_type;
   /*! \brief The device index */
-  int device_id;
+  int32_t device_id;
```

Comment:
not sure about this, it may have to match something declared elsewhere.  same with ndim.

Reply:
reverted changes in dlpack.h, after check dlpack repo, they notice these int type issues. Keep the two vars int, I guess b/c it's impactless.

FYI
https://github.com/dmlc/dlpack/issues/1
https://github.com/dmlc/dlpack/pull/8